fs/mnemofs/Kconfig

```diff
@@ -0,0 +1,18 @@
+config FS_MNEMOFS
+	bool "MNEMOFS File System"
```

Comment:
Maybe something like: "MNEMOFS NAND File System" to make it more directed to NAND, although it could be used for other medias as well